cargo-process.el

```diff
@@ -187,6 +227,26 @@ Cargo: Run the tests."
   (cargo-process--start "Test" "cargo test"))
 
 ;;;###autoload
+(defun cargo-process-current-test ()
+  "Run the Cargo test command for the current test.
+Cargo: Run the tests."
+  (interactive)
+  (let ((name (cargo-process--get-current-test)))
+    (cargo-process--start "Test" (s-concat "cargo test " name))))
```

Comment:
Use concat instead of s-concat? Would be nice to not be dependent on 's library.